src/vconsole/vconsole-setup.c

```diff
@@ -208,9 +208,9 @@ static int font_load_and_wait(const char *vc, const char *font, const char *map,
  */
 static void setup_remaining_vcs(int fd, bool utf8) {
         struct console_font_op cfo = {
-                .op = KD_FONT_OP_GET, .flags = 0,
-                .width = 32, .height = 32,
-                .charcount = 512,
+                .op = KD_FONT_OP_GET,
+                .width = UINT_MAX, .height = UINT_MAX,
+                .charcount = UINT_MAX,
```

Comment:
Yea, if any field is explicitly initialized, any other field gets 0 / null / etc.